var/spack/repos/builtin/packages/sarus/package.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import llnl.util.tty as tty
+
+from spack import *
+
+
+class Sarus(CMakePackage):
+    """Sarus is an OCI-compliant container engine for HPC systems."""
+
+    homepage = "https://github.com/eth-cscs/sarus"
+    url = "https://github.com/eth-cscs/sarus/archive/1.3.3.tar.gz"
+    git = "https://github.com/eth-cscs/sarus.git"
+    maintainers = ["Madeeks", "teonnik"]
+
+    version("develop", branch="develop")
+    version("master", branch="master")
+    version("1.4.0", tag="1.4.0")
+    version("1.3.3", tag="1.3.3")
+    version("1.3.2", tag="1.3.2")
+    version("1.3.1", tag="1.3.1")
+    version("1.3.0", tag="1.3.0")
+    version("1.2.0", tag="1.2.0")
+    version("1.1.0", tag="1.1.0")
+    version("1.0.1", tag="1.0.1")
+    version("1.0.0", tag="1.0.0")
+
+    variant(
+        "ssh",
+        default=False,
+        description="Build and install the SSH hook and custom SSH software "
+        "to enable connections inside containers",
+    )
+
+    depends_on("wget", type="build")
```

Comment:
Why wget actually?

Reply:
It's not needed, good catch! wget was necessary when tini and runc were installed from within the sarus package. However they are now separate packages in spack and are installed as proper dependencies.